code/modules/mob/living/silicon/say.dm

```diff
@@ -73,7 +73,12 @@
 	if (speaking)
 		verb = speaking.speech_verb

```

Comment:
Verb setting needs to be in the second `if(speaking)` block, after default language, or there'll be strange things happening.